packages/relay-runtime/util/ReaderNode.js

```diff
@@ -33,6 +33,7 @@ export type ReaderInlineDataFragmentSpread = {|
   +kind: 'InlineDataFragmentSpread',
   +name: string,
   +selections: $ReadOnlyArray<ReaderSelection>,
+  +args?: ?$ReadOnlyArray<ReaderArgument>,
```

Comment:
I think ideally this would be:

```suggestion
  +args: ?$ReadOnlyArray<ReaderArgument>,
```

But having it optional is a nice way to make it safe to land runtime and compiler changes separately internally. Maybe add a comment that we should come back and make the property non-optional (the value can continue to be optional).

Reply:
@captbaritone so we'll basically follow this pr up right after with the non-optional?

Reply:
Yeah, that would be perfect.

Reply:
Actually, I'll do this internally.